dsa/validate_test.go

```diff
@@ -2,23 +2,46 @@ package dsa
 
 import (
 	"encoding/json"
+	"strings"
 	"testing"
 
 	"github.com/prebid/openrtb/v20/openrtb2"
 	"github.com/prebid/prebid-server/v2/exchange/entities"
 	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	"github.com/prebid/prebid-server/v2/util/ptrutil"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestValidate(t *testing.T) {
+	validBehalf := strings.Repeat("a", 100)
+	invalidBehalf := strings.Repeat("a", 101)
+	validPaid := strings.Repeat("a", 100)
+	invalidPaid := strings.Repeat("a", 101)
+
 	tests := []struct {
 		name        string
 		giveRequest *openrtb_ext.RequestWrapper
 		giveBid     *entities.PbsOrtbBid
-		wantValid   bool
+		wantError   bool
```

Comment:
Is there a reason we are only testing the presence of the error and not the error message? We may be getting the right answer, but in an unexpected way.